sample/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
 dependencies {
     compile 'com.android.support:appcompat-v7:22.1.1'
     compile 'com.android.support:recyclerview-v7:22.1.1'
-    //compile project(':library')
-    compile 'com.prolificinteractive:material-calendarview:0.3.2'
+    compile project(':library')
+    //compile 'com.prolificinteractive:material-calendarview:0.3.2'
```

Comment:
I keep the line around for convenience when I test new releases to maven. The last release I forgot to swap it back to using the project line.